frontend/lib/src/theme/globalStyles.ts

```diff
@@ -21,7 +21,7 @@ import { EmotionTheme } from "@streamlit/lib/src/theme"
 export const globalStyles = (theme: EmotionTheme): SerializedStyles => css`
   a,
   a:visited {
-    color: ${theme.colors.primary};
+    color: ${theme.colors.linkText};
```

Comment:
global style changes always going to make me a little nervous, but it is our intent to change all links right?

Reply:
Yeah, I definitively feel the same way... The request mentions the link color in the about dialog only, but it also mentions `it should be the normal blue that we use everywhere else`, which makes me think we want all links to be the same color as the actual app links. @jrieke maybe you can shed a bit more light as to what was your thinking for this request?

Reply:
Oh sorry for the confusion. Just change the dialog, we don't need to change anything else. (I thought we already use the same color for links everywhere else).

Reply:
Makes sense! Reverted and fixed on [c30d2b4](https://github.com/streamlit/streamlit/pull/7050/commits/c30d2b4f567657dcd8b526b9dcf6ce2c16716410)